package.json

```diff
@@ -33,12 +35,10 @@
     "eslint": ">=6"
   },
   "devDependencies": {
-    "@typescript-eslint/eslint-plugin": "^2.11.0",
-    "@typescript-eslint/parser": "^2.11.0",
     "eslint": "^6.7.2",
     "typescript": "^3.7.3"
   },
   "scripts": {
-    "test": "yarn --cwd test && yarn --cwd test eslint --ext .js,.ts,.tsx ."
+    "test": "yarn --cwd test && yarn --cwd test eslint --report-unused-disable-directives --ext .js,.ts,.tsx ."
```

Comment:
Should we also include .jsx here?

Reply:
Why not, we'd need to add a jsx file as well. Sounds like a good followup